### PR TITLE
feat(appeals): add s78 LPAQ rows for eia environmental impact schedule and eia development description

### PR DESCRIPTION
**Pull request by james-hill-kc**
## Describe your changes

add s78 LPAQ rows and change pages for eia environmental impact schedule and eia development description

## Issue ticket number and link

A2-1465

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js [hunk 1]
```diff
@@ -366,7 +366,7 @@ describe('LPA Questionnaire review', () => {
 			expect(notificationBannerElementHTML).toContain('Column 2 threshold criteria status changed');
 		});
 
-		it('should render an "Environmental statement status changed" success notification banner when meets eia requires environmental statement is changed', async () => {
+		it('should render an "Environmental statement status changed" success notification banner when eia requires environmental statement is changed', async () => {
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js:369]: This was just to fix a typo in the test name

File: appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/appellant-case.test.js [hunk 2]
```diff
@@ -1620,7 +1619,7 @@ describe('appellant-case', () => {
 		});
 
 		it('should render the update due date page with pre-populated date values if there is no existing due date and applicationDecisionDate is set', async () => {
-			const decisionDate = sub(new Date(), { days: 30 }).toISOString();
+			const decisionDate = '2024-10-12T11:44:21.173Z';
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/appellant-case.test.js:1622]: This is to fix snapshots breaking due to the date value changing

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/__snapshots__/lpa-questionnaire.test.js.snap [hunk 3]
```diff
@@ -3145,6 +3169,18 @@ role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govu
 </div>"
 `;
 
+exports[`LPA Questionnaire review Notification banners should render an "Environmental statement status changed" success notification banner when eia requires environmental statement is changed 1`] = `
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/__snapshots__/lpa-questionnaire.test.js.snap:3172]: result of typo fix (see comment on associated test file)

File: appeals/web/src/server/lib/mappers/data/lpa-questionnaire/s78.js [hunk 4]
```diff
@@ -33,5 +35,7 @@ export const submaps = {
 	isAonbNationalLandscape: mapIsAonbNationalLandscape,
 	isInfrastructureLevyFormallyAdopted: mapIsInfrastructureLevyFormallyAdopted,
 	infrastructureLevyAdoptedDate: mapInfrastructureLevyAdoptedDate,
-	infrastructureLevyExpectedDate: mapInfrastructureLevyExpectedDate
+	infrastructureLevyExpectedDate: mapInfrastructureLevyExpectedDate,
```
Comment (Si3rr4wow) [inline appeals/web/src/server/lib/mappers/data/lpa-questionnaire/s78.js:38]: can we turn comma dangle back on 😩

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/eia-environmental-impact-schedule/eia-environmental-impact-schedule.controller.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+} from './eia-environmental-impact-schedule.mapper.js';
+import { changeEiaEnvironmentalImpactSchedule } from './eia-environmental-impact-schedule.service.js';
+import { getLpaQuestionnaireFromId } from '../lpa-questionnaire.service.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+export const getChangeEiaEnvironmentalImpactSchedule = async (request, response) => {
```
Comment (Si3rr4wow) [inline appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/eia-environmental-impact-schedule/eia-environmental-impact-schedule.controller.js:14]: Is this just an alias?

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/eia-development-description/eia-development-description.controller.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+import { changeEiaDevelopmentDescriptionPage } from './eia-development-description.mapper.js';
+import { changeEiaDevelopmentDescription } from './eia-development-description.service.js';
+import { getLpaQuestionnaireFromId } from '../lpa-questionnaire.service.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+export const getChangeEiaDevelopmentDescription = async (request, response) => {
```
Comment (Si3rr4wow) [inline appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/eia-development-description/eia-development-description.controller.js:11]: again just an alias?

**Review by Si3rr4wow — DISMISSED** [review]

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/eia-environmental-impact-schedule/eia-environmental-impact-schedule.controller.js (hunk 5 above)
Comment (james-hill-kc) [reply appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/eia-environmental-impact-schedule/eia-environmental-impact-schedule.controller.js]: Yep - refactored to reference `renderChangeEiaEnvironmentalImpactSchedule` directly now

File: appeals/web/src/server/lib/mappers/data/lpa-questionnaire/s78.js (hunk 4 above)
Comment (james-hill-kc) [reply appeals/web/src/server/lib/mappers/data/lpa-questionnaire/s78.js:38]: Is that an eslint config change, or something in github?

Comment (Si3rr4wow) [reply appeals/web/src/server/lib/mappers/data/lpa-questionnaire/s78.js:38]: It just means lines like this already have a comma so the diff is shorter when you add to an object or array

Comment (james-hill-kc) [reply appeals/web/src/server/lib/mappers/data/lpa-questionnaire/s78.js:38]: It would be nice not to have these in the diff!

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**